### DISCUSSION TITLE
Add an option to avoid sharing participant emails in finalized calendar hit

### DISCUSSION THREAD

**Discussion by charles-m-knox** [lukevella/rallly#1207] [Ideas]
As discussed in my PR #1205 , I want the ability to hide the list of attendees from the finalized calendar hit. I lead a few organizations and use Rally to gather consensus for 100+ people at times. My users have not consented to share their email with other users.

I implemented this successfully in #1205 and tested it, but it was closed by @lukevella on the grounds it doesn't fit his design parameters.

@lukevella please discuss your thoughts with me in this thread. Thank you

**Comment by charles-m-knox** [comment]
I added it as a poll setting because some polls only have a handful of attendees (small group meetings can have emails shared usually), whereas other polls have many participants.

For example: in a recent large poll, I unintentionally shared every participant's email with all other participants, which I think is generally not the desired behavior for larger polls.

Therefore, the finalized ICS calendar hit should be configurable to show or hide everyone who responded.

I think adding this in the poll creation settings is sensible. I'd be curious to hear @lukevella 's perspective, perhaps there's something I overlooked.

**Comment by lukevella** [comment]
Hi @charles-m-knox, I think if this were to be implemented it would make a lot more sense to be part of the Finalize dialog. I'm not sure how this can be effectively communicated to the user and this is the first time I'm hearing about such a request. You might be right about users wanting this but a single data-point is not enough to start investing time into something which I hope you will understand.

I'm moving this to discussions as it's not something that will likely be addressed soon.

**Reply by charles-m-knox** [reply]
Hi @lukevella I appreciate your feedback. I'm open to waiting until you hear from other folks that might ask for this feature. Gathering data before making decisions is always important.

I'd encourage you to consider a couple things:

- The work is already done in my PR. It works as-is.
- I am getting a slight sense of analysis paralysis from you. I don't think you need to overthink it.
- If you're uncertain how to proceed, we could implement it as an experimental feature, and have an env flag that exposes the field.
- We can add a question mark bubble with a link to documentation on the feature, which I'd be happy to contribute.

Looking forward to hearing your thoughts. Thank you

**Reply by chrisliebaer** [reply]
I'm honestly disappointed that this feature was declied. I've used ralllye a few times to organiz events with friends and after first doxxing everyones email by accident, I told everyone not to enter an e-mail address and sent out a manual update after finalization. There is always a conflict between sharing the email you actually care to read and having others get to know the email you do actually read.

Including this feature inside the poll settings also prevents forgetting about it before hitting send. So I think it's the perfect spot, although I would actually prefer if this was an instance wide-default.

**Reply by charles-m-knox** [reply]
> I'm honestly disappointed that this feature was declied. I've used ralllye a few times to organiz events with friends and after first doxxing everyones email by accident, I told everyone not to enter an e-mail address and sent out a manual update after finalization. There is always a conflict between sharing the email you actually care to read and having others get to know the email you do actually read.
> 
> Including this feature inside the poll settings also prevents forgetting about it before hitting send. So I think it's the perfect spot, although I would actually prefer if this was an instance wide-default.

Members of the community that take the time to voice their needs (such as yourself) will continue to help provide Luke with the data he needs to implement the feature. Perhaps some day enough people will chime in. Thanks for sharing your perspective @chrisliebaer
